yarn.lock

```diff
@@ -6398,6 +6398,19 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@jest/types@npm:^27.2.5":
```

Comment:
Did we upgrade jest in this PR as well? If not, should we roll back this to the 26 version?